common/test/acceptance/pages/lms/track_selection.py

```diff
@@ -49,7 +49,7 @@ def enroll(self, mode="audit"):
         if mode == "verified":
             # Check the first contribution option, then click the enroll button
             self.q(css=".contribution-option > input").first.click()
-            self.q(css="input[name='verified_mode']").click()
+            self.q(css="button[name='verified_mode']").click()
```

Comment:
I wonder if we have an running or upcoming experiments that are looking for an _input_ (which isn't to say that we shouldn't make this change, just that maybe we should check first).

Reply:
hm, good point we probably do.  
i'll add an action item